crates/cargo-wdk/src/actions/build/mod.rs

```diff
@@ -414,4 +400,118 @@ impl<'a> BuildAction<'a> {
         info!("Finished building {package_name}");
         Ok(())
     }
+
+    // Determines the target directory to hand to the packaging task by
+    // scanning `cargo build --message-format=json` output and locating
+    // the cdylib artifact produced for this package (normalized name and
+    // manifest path must match). The parent folder of that `.dll` path is
+    // returned once resolved to an absolute path.
+    // Errors if the cdylib artifact cannot be found or its parent cannot
+    // be made absolute.
```

Comment:
The comment says "name and manifest path must match", but we check only package ID now.